code/game/machinery/kitchen/cooking_machines/_cooker_output.dm

```diff
@@ -34,7 +34,7 @@
 	M.Scale(scale)
 	src.transform = M
 
-	w_class *= scale
+	w_class = round(w_class * scale)
```

Comment:
Is this an undocumented change/bugfix or is it related to one of the other fixes mentioned?

Reply:
It's related to custom output food not fitting in containers. Apparently, if w_class is set to anything other than a whole number, it simply won't fit into any container.